modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -135,7 +136,19 @@ const EditorUpload = function (editor: Editor): EditorUpload {
             blobCache.removeByUri(image.src);
             replaceImageUriInView(image, uploadInfo.url);
           } else if (uploadInfo.error) {
-            ErrorReporter.uploadError(editor, uploadInfo.error);
+            if (uploadInfo.error.options.remove) {
+              if (Rtc.isRtc(editor)) {
+                // To be replaced by RTC API to mirror DOM changes when such is implemented.
+                console.log('Could not erase image due to RTC plugin'); // eslint-disable-line no-console
```

Comment:
Instead of this, how about rewording this to be something like this?
```
Removing images on failed uploads is currently unsupported for RTC
```
It just clarifies it's just an interim thing, rather than sounding like it'll never work.

Reply:
Also should this be a `console.warning` or `console.error`? `console.log` is unlikely to be the logging level we want here.